scripts/clearml_agent/clearml.conf

```diff
@@ -137,12 +140,22 @@ agent {
 
     # optional arguments to pass to docker image
     # these are local for this agent and will not be updated in the experiment's docker_cmd section
-    extra_docker_arguments: ["-v","G:\Shared drives\AQUA:/data/aqua-ml-data","--env","SIL_NLP_DATA_PATH=/data/aqua-ml-data", "-v", "C:\Users\johnm\AppData\Local\pypoetry\Cache:/root/.cache/pypoetry"]
+    extra_docker_arguments: ["--env","SIL_NLP_DATA_PATH=/aqua-ml-data",
+        "--env","AWS_REGION=us-east-1",
+        "--env","AWS_ACCESS_KEY_ID=*********",
+        "--env","AWS_SECRET_ACCESS_KEY=***************",
+        "-v","C:\Users\johnm\AppData\Local\pypoetry\Cache:/root/.cache/pypoetry",
```

Comment:
This path is harcoded to a local path on your machine. Is this an issue?

Reply:
No - it is just an example, as the access keys are as well.